src/Movie/Movie.js

```diff
@@ -1,49 +1,83 @@
 import React, { Component } from "react";
 import "./Movie.css";
 import Videos from "../Videos/Videos";
+import PropTypes from 'prop-types'
 
+class Movie extends Component {
+  constructor( props ) {
+    super(props)
+    this.state = {
+      currentMovie: [],
+      isLoading: false
+    }
+  }
+  individualMovieFetchCall = (id) => {
```

Comment:
Love changing movie to a class component so it can hold its own state. Well done!